backend/workflow_manager/workflow_v2/execution.py

```diff
@@ -404,15 +404,20 @@ def update_execution_err(execution_id: str, err_msg: str = "") -> WorkflowExecut
     @staticmethod
     def update_execution_task(execution_id: str, task_id: str) -> None:
         try:
-            if not task_id:
+            if not task_id or task_id == "":
                 logger.warning(
-                    f"task_id: '{task_id}' is NULL / empty for "
-                    f"execution_id: {execution_id}, expected to have a UUID"
+                    f"Skipping empty task_id: '{task_id}' for "
+                    f"execution_id: {execution_id}, expected to have a valid UUID"
```

Comment:
```suggestion
                    f"Skipped setting taskID for execution {execution_id} since its NULL/empty"
```
NIT: Printing the empty task ID doesn't make sense, do fix line-length issues